packages/spec/integration/pwa/__tests__/build.js

```diff
@@ -0,0 +1,37 @@
+describe('pwa production build', () => {
+  let url;
+
+  beforeAll(async () => {
+    jest.setTimeout(30000);
+    await HopsCLI.build();
+    url = await HopsCLI.serve();
+  });
+
+  it('registers a service worker', async () => {
+    const { page } = await createPage();
+    await page.goto(url, { waitUntil: 'networkidle2' });
+
+    // wait until service worker is installed
```

Comment:
Wonder if it could make sense to check that the resources are *not* served from the service worker at the first request to exclude potential false positives.